.github/workflows/release.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     - name: Update SpecsTesting repo setup
       run: |
          ossbotaccess=`cat oss-bot-access.txt`
-         BOT_TOKEN="${ossbotaccess}" nightly_test_version="${nightly_version}" scripts/release_testing_setup.sh
+         BOT_TOKEN="${ossbotaccess}" test_version="${nightly_version}" scripts/release_testing_setup.sh nightly_testing
```

Comment:
Where is test_version used?

Reply:
It is used in release_testing_setup.sh to update a tag.
https://github.com/firebase/firebase-ios-sdk/blob/38af2d564336b06efdd24920de08230364d4957b/scripts/release_testing_setup.sh#L31-L35

Reply:
test_version should be updated in the workflow and testing tags need to be removed manually from repo at this moment.